portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,77 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Nicole's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Nicole's Portfolio</h1>
+        <div id = "gallery">
```

Comment:
In order to be consistent with the rest of the code, let's remove the whitespaces between the equal sign:
`<div id="gallery">`.

Reply:
deleted whitespace.